client/src/app/primitives/modal/Modal.js

```diff
@@ -9,30 +9,40 @@
  */
 
 import React from 'react';
+import ReactDOM from 'react-dom';
 
 import classNames from 'classnames';
 
-import css from './ModalWrapper.less';
+import css from './Modal.less';
 
 
-const ModalWrapper = props => {
-  const handleBackgroundClick = event => {
+function Modal(props) {
```

Comment:
Is this the right moment to introduce React hooks into our code base? I'm not sure about this one.

Reply:
We've bumped React version to 16.8 about 4 months ago and it seems the hooks are not going anywhere. Why should we bother?

Reply:
Anyway, the class alternative is not overly verbose or anything. Let's leave hooks out of this PR.

Reply:
And yes, the `useRef` usage looked a bit clumsy, so it's not only about not hurting but the actual benefit of removing hooks out of this ;)